ui/templates/includes/resource_panel.html

```diff
@@ -11,6 +11,11 @@
             <span class="hidden-xs">XML</span>
           </a>
         </li>
+        <li>
```

Comment:
I'm thinking you can leave this in, but leave it commented out.  Also, the tab looks funny due to the words stacking on top of each other, so we will probably want to change it to just `Assets` or `Static`, or `Files` maybe.  Either that or tweak the CSS so `Static Assets` shows on one line.